NewPlatform.Flexberry.ServiceBus.IntegratedTests/Components/ObjectRepository/CachedDataServiceObjectRepositoryTest.cs

```diff
@@ -187,7 +213,6 @@ public void TestGetRestrictionsForMsgType()
             Func<IEnumerable<TDataObjectType>> methodUnderTest,
             int updatePeriod,
             string[] propertiesToCheck)
-            where TDataObjectType : DataObject

```

Comment:
The `IObjectRepository.GetAllClients` method should return an array of `Client` objects and not `ServiceBusClient`.